src/crypto/crypto_context.cc

```diff
@@ -812,6 +815,58 @@ void GetRootCertificates(const FunctionCallbackInfo<Value>& args) {
       Array::New(env->isolate(), result, arraysize(root_certs)));
 }
 
+MaybeLocal<Array> X509sToArrayOfStrings(Environment* env,
+                                        const std::vector<X509*>& certs) {
```

Comment:
Since you're using `X509View` here anyway, it would be nice to move away from the raw pointers in this std::vector. Can we make this an `std::vector<ncrypto::X509View>&` ?

Reply:
I've considered this but it seems a bit too complicated for this PR, because we get the raw pointers from the function-local static storage for thread safety. Passing in `std::vector<ncrypto::X509View>` means doing another loop to convert them into vectors of  X509View first, and then this must be called for all the caller before `X509sToArrayOfStrings`. It seems like a lot of extra code/slight overhead for no particular gains.